README.md

```diff
@@ -77,6 +77,6 @@ All variable key names used to extract data from debug launch configurations can
 
 The following list outlines the setting names and default values:
 
-- `svd-viewer.svdPathConfig` - Debug configuration key to use to get the SVD path (default `svdPath`)
-- `svd-viewer.deviceConfig` - Debug configuration key to use to get the device name (default: `deviceName`)
-- `svd-viewer.processorConfig` - Debug configuration key to use to get the processor name (default: `processorName`)
+- `peripheral-inspector.svdPathConfig` - Debug configuration key to use to get the SVD path (default `svdPath`)
```

Comment:
Hi @thegecko, do you think renaming `svdPathConfig` to something more general like `peripheralDefinitionPathConfig` and `svdPath` to `peripheralDefinitionPath` ?

Perhaps it requires other code changes too, so, it can also be left to another update.

Reply:
Renamed this to `definitionPath` and `definitionPathConfig`